nncf/torch/quantization/layers.py

```diff
@@ -52,6 +52,45 @@ def from_str(config_value: str) -> 'HWConfigType':
         raise RuntimeError("Unknown quantizer ONNX export mode string")
 
 
+class PTQuantizerConfig(QuantizerConfig):
+    def __init__(self, num_bits: int = 8,
+                 mode: QuantizationMode = QuantizationMode.SYMMETRIC,
+                 signedness_to_force: Optional[bool] = None,
+                 per_channel: bool = False,
+                 half_range=None,
+                 narrow_range=None,
+                 logarithm_scale=None):
+        super().__init__(num_bits, mode, signedness_to_force, per_channel)
+        self.half_range = half_range
+        self.narrow_range = narrow_range
+        self.logarithm_scale = logarithm_scale
+
+    @classmethod
+    def from_qconfig(cls, qconfig: QuantizerConfig, half_range, narrow_range, logarithm_scale):
+        return cls(qconfig.num_bits,
+                   qconfig.mode,
+                   qconfig.signedness_to_force,
+                   qconfig.per_channel,
+                   half_range,
+                   narrow_range,
+                   logarithm_scale)
+
+    def get_state(self) -> Dict[str, Any]:
```

Comment:
if you use `get_state`, there should be `from_state` or `load_state` on loading compression state.